### PR TITLE
Top level key details dialog

### PR DESCRIPTION
**Pull request by mankyKitty**
Skeleton for top level key details dialog, will flesh out as types & designs arrive.

### CODE REVIEW

**Comment by mankyKitty** [comment]
On top of #344 as it requires the new account / key types.

File: common/src/Common/Wallet.hs [hunk 1]
```diff
@@ -405,6 +405,11 @@ lookupAccountRef (Some ref) accounts = case ref of
 accountUnfinishedCrossChainTransfer :: Account -> Maybe UnfinishedCrossChainTransfer
 accountUnfinishedCrossChainTransfer = _accountInfo_unfinishedCrossChainTransfer . accountInfo
 
+accountBalance :: Account -> Maybe AccountBalance
+accountBalance (r :=> Identity a) = _accountInfo_balance $ case r of
```
Comment (alexfmpe) [inline common/src/Common/Wallet.hs]: `_accountInfo_balance . accountInfo`

File: frontend/src/Frontend/UI/Dialogs/KeyDetails.hs [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  => model
+  -> Dynamic t (Map ChainId NonVanityAccount)
+  -> m mConf
+uiChainTable model keyChainInfo = do
+  let
+    tableAttrs =
+      "style" =: "table-layout: fixed; width: 98%"
+      <> "class" =: "key-details table"
+  _events <- elAttr "table" tableAttrs $ do
+    el "colgroup" $ do
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Dialogs/KeyDetails.hs:103]: 110%?

File: frontend/src/Frontend/UI/Dialogs/KeyDetails.hs [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  :: ( DomBuilder t m
+     , PostBuild t m
+     )
+  => model
+  -> ChainId
+  -> Dynamic t NonVanityAccount
+  -> m (Event t ())
+uiChainTableRow _model chain dNonVanity = do
+  let
+    keyDeetCls = mappend "key-details__"
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Dialogs/KeyDetails.hs]: Extra 'e'?

File: frontend/src/Frontend/UI/Dialogs/KeyDetails.hs [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+    showBal (Just b) = tshow b <> " KDA"
```
Comment (alexfmpe) [inline frontend/src/Frontend/UI/Dialogs/KeyDetails.hs]: Are we intentionally not indicating pending transfers here? See https://github.com/kadena-io/chainweaver/blob/41538c17dc912233ff1c301a3110839d43d00d0c/frontend/src/Frontend/UI/Widgets.hs#L556

Comment (mankyKitty) [reply frontend/src/Frontend/UI/Dialogs/KeyDetails.hs]: I had completely forgotten about that, thanks!!

File: frontend/src/Frontend/Wallet.hs [hunk 5]
```diff
@@ -83,8 +83,8 @@ import Frontend.KadenaAddress
 import Frontend.Storage
 import Frontend.Network
 
-accountIsCreated :: Account -> AccountCreated
```
Comment (alexfmpe) [inline frontend/src/Frontend/Wallet.hs:86]: Give it back!

**Pull request merged**